packages/web3-core-method/src/index.js

```diff
@@ -436,16 +438,18 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
                                         // If send was raw, fetch the transaction and reconstitute the
                                         // original params so they can be replayed with `eth_call`
                                         if (method.call === 'eth_sendRawTransaction'){
-                                            var txToReplay = await _ethereumCall.getTransactionByHash(receipt.transactionHash);
-
-                                            txReplayOptions = formatters.inputTransactionFormatter({
-                                                data: txToReplay.input,
-                                                to: txToReplay.to,
-                                                from: txToReplay.from,
-                                                gas: txToReplay.gas,
-                                                gasPrice: txToReplay.gasPrice,
-                                                value: txToReplay.value
-                                            })
+                                            var rawTransactionHex = payload.params[0];
+
+                                            var parsedTx = EthersTransactionUtils.parse(rawTransactionHex);
```

Comment:
Removed the inputTransactionFormatter function because ethersproject/Transactions already handles address checksums. Value, gasPrice are converted to hex str from BigNumber.